backend/main.py

```diff
@@ -268,6 +271,54 @@ def update_leader_board_total(session: Session) -> None:
         logger.exception("Error during user states update (total)")
 
 
+@app.on_event("startup")
+def set_startup_time() -> None:
+    global startup_time
+    startup_time = utcnow()
+
+
+def update_all_user_streak(session: Session) -> None:
+    statement = select(User)
+    result = session.exec(statement).all()
```

Comment:
Not sure what this is supposed to do? returns all users by type annotation says None? The name says update but nothing is updated.